container error

Thank you very much for this release!

By create the container, I get an error. 
(The image I have only built without font configuration)
How can I resolve this?
```
root@DS:~# docker run -it \
>     -p 8080:8080 \
>     hc-pdf-server:latest
yarn run v1.22.5
$ node dist/server.js
node:internal/modules/cjs/loader:928
  throw err;
  ^

Error: Cannot find module '/app/dist/server.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:925:15)
    at Function.Module._load (node:internal/modules/cjs/loader:769:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
root@DS:~# 
```

--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
   "scripts": {
     "build": "tsc",
     "test": "tap --timeout=180 test/**/*.test.ts",
-    "start": "node dist/server.js",
+    "start": "node dist/src/server.js",
     "dev": "ts-node ./src/server.ts",
     "lint": "eslint --fix ./ --ext ts"
   },

